docs/troubleshooting/performance.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Performance Issues"
+date: 2023-08-02
+summary: "Steps on how to diagnose performance bottlenecks in PCSX2"
+draft: false
+toc: true
+sidebar_position: 2
+---
+
+## It works but it's slow?
+
+This is the most common problem that users experience. PCSX2 is a very hardware intensive program as with any emulators in general, especially on your processor.
+
+:::info
+It is highly recommended you read the first post of this thread: [Will PCSX2 run fast on my computer?](https://forums.pcsx2.net/Thread-Sticky-Will-PCSX2-run-fast-on-my-computer) or refer to the Requirements page and if you still have questions reply to the thread or in the Discord, there are many helpful members who can help with your performance issue.
+:::
+
+## Enable OSD to measure performance bottleneck
+
+Whenever you're reporting for perfomance issues, please post a screenshot of the performance counters in the PCSX2 On Screen Display (OSD).
+
+### Where do I enable the counters?
+
+`Settings > Graphics > OSD`
+
+### What boxes do I tick?
+
+For best results, please tick all the boxes. This will add a lot of things to the screen, but you can turn them back off once we are done looking at the performance statistics.
+
+### Why do you need this?
+
+These numbers help us identify where performance bottlenecks are in your system. If you are having performance issues, please make sure you screenshot the numbers while the game is running poorly.
+
+<Image cols={7} src={require("./img/OSD.webp").default} />
+
+## OSD Legends
+
+The OSD contains a lot of acronyms and abbreviations which can be confusing to understand, so here is a list of ones which are available.
+
+### Statistic Metrics
+
+This statistics shows all kind of reports in regards to PCSX2's internal performance reporting. Usually located on the top right.
+
+:::tip
+This metrics is crucial in identifying performance bottlenecks in PCSX2.
+:::
+
+#### Hardware Mode
+
+```s
+HC  = Hash Cache size (in MB)
```

Comment:
`s` is a weird language for syntax highlighting, i dont even know that it is.  Since this is `key=value` you might find `ini` to look nicer.

Reply:
Thanks, i'm just not sure what language to use